examples/chef/devices/rootnode_basicvideoplayer_0ff86e943b.zap

```diff
@@ -2444,7 +2444,7 @@
               "side": "server",
               "type": "boolean",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "NVM",
```

Comment:
This is a very coarse way to force a value. It makes it stateful to last value set. I recommend against doing this in ZAP and rather keeping state in the application. This at worst could cause some attributes that are not marked `N` in the spec (like `OnOff`) to be stateful across binary runs, which is likely not expected by some chef users.

Reply:
> This at worst could cause some attributes that are not marked N in the spec (like OnOff) to be stateful across binary runs

Only few attributes need to be kept  be during running cycle will be marked as NVM. For example, I have been asked many times that why the adjusted the level value gone when they rerun the application without factory reset?